llvm/lib/Support/RISCVISAInfo.cpp

```diff
@@ -106,6 +106,8 @@ static const RISCVSupportedExtension SupportedExtensions[] = {
 
     {"zdinx", RISCVExtensionVersion{1, 0}},
 
+    {"zexperimental", RISCVExtensionVersion{1, 0}},
+
```

Comment:
I guess we don't really need this dummy extension in RISCVISAInfo?

Reply:
No, we don't~